src/lib/_imports/trumps/u-highlight/u-highlight.demo.css

```diff
@@ -0,0 +1,7 @@
+.u-highlight {
+    padding-inline: var(--tag-padding);
```

Comment:
The `--tag-padding` does not exist now. Just put some small value in there, so user can see that the padding has effect a different effect than were they to not use `u-highlight`.